docs/docs/extensions/virtuallistview.md

```diff
@@ -138,54 +138,54 @@ interface VirtualListViewItemInfo {
 ```
 
 ## Props
-``` javascript
-// Should the list animate additions, removals and moves within the list?
-animateChanges?: boolean;
-
+``` javascript    testId?: string;
 // Ordered list of descriptors for items to display in the list.
-itemList: VirtualListViewItemInfo[];
+itemList: ItemInfo[];
 
-// Logging callback to debug issues related to the VirtualListView.
-logInfo?: (textToLog: string) => void;
+// Callback for rendering item when it becomes visible within view port.
+renderItem: (renderDetails: VirtualListCellRenderDetails<ItemInfo>) => JSX.Element | JSX.Element[];
```

Comment:
What is the max line length?

Reply:
Good question. I'm not sure if it's a set amount. I guess it would be a fixed max length given that we're using a mono-spaced font. I just know that it overflows and makes the docs difficult to read if the lines get too long.